components/page-header/page-header.less

```diff
@@ -57,11 +57,6 @@
 	.page-header-small & {
```

Comment:
Looks fine, but whitespace is messy (probably throughout the project, but let's fix 1 file at a time)

replace all `····` with `⭾` :)